src/components/AiMarketplace/MainSection/ServiceCollection/StyledPagination/PaginationMenus.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import MenuItem from "@material-ui/core/MenuItem";
+import OutlinedInput from "@material-ui/core/OutlinedInput";
+import Select from "@material-ui/core/Select";
+
+const PaginationMenus = ({ totalItems, itemsPerPage, handleItemsPerPage, classes }) => {
+  const menus = [];
+  const step = 10;
+
+  if (totalItems >= step) {
```

Comment:
Do we need to give all the options for items per page ? I think we should limit it to some number. Else we will end up having n/10 items per page as options which is not the standard practice I believe. Also we could just give the predefined options without worrying about the total number of items.

Reply:
Yh valid point.. That's y I have abstracted step as a separate variable, so we could discuss on it..